### PR TITLE
Hackathon

### LINKED ISSUE #22: Gotta go hack specific doc

### PR DESCRIPTION
**Pull request by qdrn**
Closes #22

### CODE REVIEW

File: hackathon.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+============
+
+First of all you need to have a client to interact with the network.
+We recommend to use the pre-built binairies available `here <https://github.com/massalabs/massa/releases>`_.
+
+The zip will contains two folders one called `massa-node` and the other `massa-client`.
+For this hackathon you will only need the client. In the folder `massa-client` you will
+find an executable `massa-client` that will be your client for the whole Hackathon.
+
+In the `massa-client/config` folder create a `config.toml` file with the following content:
```
Comment (longgamma-massa) [inline hackathon.rst:18]:
```suggestion
In the `massa-client/config` folder, create a `config.toml` file with the following content:
```

File: hackathon.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+.. code-block::
+    [default_node]
+    ip = "141.94.218.103"
+
+This will allow your client to connect to a remote node run by the Massa team
+so that you don't have to run a node yourself.
```
Comment (longgamma-massa) [inline hackathon.rst:25]:
```suggestion
so that you don't have to run a node by yourself.
```

File: hackathon.rst [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+Now that you have created a wallet you can check the address, public key, balance, etc... with this command:
+
+.. code-block::
+
+    wallet_info
+
+To develop and send smart-contracts to the blockchain you will need some tokens.
+You can get some by sending your address to the faucet bot in the "testnet-faucet"
+channel of our `Discord <https://discord.com/invite/massa>`_, or if you don't have
+Discord ask us and we'll happily send you some.
```
Comment (longgamma-massa) [inline hackathon.rst:42]:
```suggestion
Discord make sure to tell us and we'll happily send you some.
```

File: hackathon.rst [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+Smart contracts
+===============
+
+You'll find here relevant links to ressources for smart-contract development.
+
+Setting up your working environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :ref:`Getting started tutorial <sc-getting-started>` will get you through all the
+steps to setup your environment for smart-contract development and show you how to execute
```
Comment (longgamma-massa) [inline hackathon.rst:56]:
```suggestion
steps to set up your environment for smart-contract development, and show you how to execute
```

File: hackathon.rst [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+a simple hello world smart-contract.
```
Comment (longgamma-massa) [inline hackathon.rst:57]:
```suggestion
a simple "Hello world" smart-contract.
```

File: hackathon.rst [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+Massa's allow you to store 
+
+To interact with smart-contracts from your decentralized application we recommend
+to use our `massa-web3 <https://github.com/massalabs/massa-web3>`_ library.
+
+You'll find in the `create-react-app-massa <https://github.com/massalabs/create-react-app-massa>`_
+repo a React template that is compatible with our massa-web3 library.
+
+You'll find several examples of frontend developped using the massa-web3 library or in pure JS
```
Comment (longgamma-massa) [inline hackathon.rst:81]:
```suggestion
You'll find several examples of the frontend developed using the massa-web3 library or in pure JS
```

**Review by longgamma-massa — CHANGES_REQUESTED** [review]
Overall comment: I think the proper way to name smart contracts is "smart contract" not "smart-contracts". When you want to say development of SC, then you use "smart-contract development". My other suggestions are given for each typo

File: hackathon.rst (hunk 6 above)
Comment (longgamma-massa) [reply hackathon.rst]: I still see 2 unresolved changes @qdrn

Comment (qdrn) [reply hackathon.rst]: It's outdated. I did not implement directly the suggestion. You have to go through the page to see if it's ok.

**Comment by qdrn** [comment]
I've added the ending sentence @longgamma-massa. We should also mention Thyra doc when it's ready. I'll make a followup issue.

**Pull request merged**